module/zfs/zfs_ioctl.c

```diff
@@ -2326,8 +2327,23 @@ zfs_ioc_dataset_list_next(zfs_cmd_t *zc)
 static int
 zfs_ioc_snapshot_list_next(zfs_cmd_t *zc)
 {
-	objset_t *os;
 	int error;
+	objset_t *os, *ossnap;
+	dsl_dataset_t *ds;
+	uint64_t skip_after = 0, skip_to = 0;
+
+	if (zc->zc_nvlist_src_size != 0) {
+		nvlist_t *props = NULL;
+		error = get_nvlist(zc->zc_nvlist_src, zc->zc_nvlist_src_size,
+		    zc->zc_iflags, &props);
+		if (error != 0)
+			return (error);
+		(void) nvlist_lookup_uint64(props, SNAP_ITER_MIN_TXG,
+		    &skip_to);
```

Comment:
let's change the variable names here to match the new `min/max txg` nomenclature.